content/docs/hooks-faq.md

```diff
@@ -90,23 +90,23 @@ Hooki same w sobie, posiadają pewną krzywą uczenia się. Jeżeli brakuje czeg
 
 Jeżeli jesteś gotowy, zachęcamy do spróbowania Hooków w nowych komponentach, które piszesz. Upewnij się że wszyscy z twojego zespołu, wiedzą jak ich używać i są zapoznani z tą dokumentacją. Nie zalecamy przepisywania twoich istniejących klas do Hooków, chyba że z jakiegoś powodu i tak planowałeś je przepisać (na przykład aby naprawić błędy).
```

Comment:
Zachęcamy do wypróbowania hooków w nowych komponentach. Upewnij się, że wszyscy z twojego zespołu wiedzą, jak ich używać i są zapoznani z tą dokumentacją. Nie zalecamy przepisywania istniejących klas do hooków, chyba że z jakiegoś powodu i tak mieliście to w planach (na przykład, żeby naprawić błędy).